addon/components/o-s-s/progress-bar.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+type ProgressBarSkins = 'primary' | 'warning' | 'success';
+
+interface OSSProgressBarArgs {
+  value: number;
+  label?: string;
+  valueIsVisible?: boolean;
+  skin?: ProgressBarSkins;
+  small?: boolean;
```

Comment:
Was this asked by William ?

Reply:
The sync with william determined that they didn't want any real changes to it . The idea was only to mimic the one in prod in OSS . so I got that from what was specified in the suggested design on linear & what was done for upf-progress. @skin & @small are in use on the soft so they went in. @label & @labelIsVisible are in use in the sidepanel

Reply:
I think we should take the occasion to make this way simpler than it is currently then iterate instead of copying all of the current features.

For example, prefer a `size="sm"` arg instead of `small=true` arg here. I'm the one who did the old component and i clearly regretted that choice in some other component because you can end up doing something like: `large={{true}}` but nothing prevents you from doing `small={{true}} large={{true}}` together